### DISCUSSION TITLE
classes/class_animationnodeblend3

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#88] [User-contributed notes]
# classes/class_animationnodeblend3

Inherits: AnimationNodeSync< AnimationNode< Resource< RefCounted< Object Blends two of three animations linearly inside of an AnimationNodeBlendTree. Description: A resource to add to an AnimationN...

https://docs.godotengine.org/en/stable/classes/class_animationnodeblend3.html

**Comment by p-planet** [comment]
Use the [remap()](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html#class-globalscope-method-remap) method with a [clamp](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html#class-globalscope-method-clampf) to assign a variable to this node.

**Comment by Danikakes** [comment]
How does this not have the filter?

**Reply by Calinou** [reply]
Are you referring to https://github.com/godotengine/godot-proposals/issues/8259 or https://github.com/godotengine/godot-proposals/issues/8262?
